CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.16) # Qt requires CMake 3.16
+﻿cmake_minimum_required(VERSION 3.16) # Qt requires CMake 3.16
```

Comment:
Don't ask me what changed here. I don't know and I'm not aware that I changed anything there.